### ISSUE TITLE
Training result using custom dataset is poor compared to the previous official version

### ISSUE THREAD

**Issue by XiaoxueZang** [tensorflow/nmt#177]
I tried to use this new seq-to-seq model to train my private dataset but I found that the training result of this new model is poor compared to the previous public version (https://www.tensorflow.org/versions/r1.0/tutorials/seq2seq). I have made all the parameters of the two models to be the same and used SGD algorithms with learning rate 0.5 (because if I use 1, the loss explodes during training) and I got only an accuracy of around 15% for this model and an accuracy of 72% for the old model. I also tried the parameters you recommend but still got an accuracy of around 13%. I wonder if there is any difference between these two versions and I think the gap should not be this big. I wonder what could possibly go wrong.
